roles/tas_single_node/tasks/certificates.yml

```diff
@@ -245,13 +245,26 @@
         (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_tsa_certificate_chain) | list | length) == 0
         and tas_single_node_signer_type == 'file'
 
+
+    - name: Create temporary directory for manipulating CA certificate chain
+      ansible.builtin.tempfile:
+        state: directory
+        suffix: tmp_dir
+      register: tmpdir
+      changed_when: false
+
     - name: Combine leaf and root CA certificates into a chain
       ansible.builtin.command: >
         /bin/bash -c '
-        cat {{ tas_single_node_remote_tsa_leaf_certificate }} {{ tas_single_node_remote_tsa_certificate_chain }} > /tmp/certificate-chain.tmp &&
-        mv /tmp/certificate-chain.tmp {{ tas_single_node_remote_tsa_certificate_chain }};'
+        cat {{ tas_single_node_remote_tsa_leaf_certificate }} {{ tas_single_node_remote_tsa_certificate_chain }} > {{ tmpdir.path }}/certificate-chain.tmp &&
+        mv {{ tmpdir.path }}/certificate-chain.tmp {{ tas_single_node_remote_tsa_certificate_chain }};'
       register: result
       changed_when: "'changed' in result.stdout"
       when: >
         (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_tsa_certificate_chain) | list | length) == 0
         and tas_single_node_signer_type == 'file'
+
+    - name: Delete content & directory
+      ansible.builtin.file:
+        state: absent
+        path: tmpdir.path
```

Comment:
I think this is incorrect. This should be `{{ tmpdir.path }}` (and you'll find out if that's correct by the idempotence check failing, as this directory will always be present - you'll need to use `changed_when: false` as well here).

Reply:
Ok will make that change